content/docs/3.0/faq/faq.md

```diff
@@ -0,0 +1,166 @@
+
+## Does TiKV have ACID semantics?
+
+Yes. ACID semantics are guaranteed in TiKV:
+
+* Atomicity: Each transaction in TiKV is "all or nothing": if one part of the transaction fails, then the entire transaction fails, and the database state is left unchanged. TiKV guarantees atomicity in each and every situation, including power failures, errors, and crashes. 
+* Consistency: TiKV ensures that any transaction brings the database from one valid state to another. Any data written to the TiKV database must be valid according to all defined rules.
+* Isolation: TiKV provides snapshot isolation (SI), snapshot isolation with lock, and externally consistent reads and writes in distributed transactions.
+* Durability: TiKV allows a collection of machines to work as a coherent group that can survive the failures of some of its members. So in TiKV, once a transaction has been committed, it will remain so, even in the event of power loss, crashes, or errors. 
+
+## How are transactions in TiKV lock-free?
+
+TiKV has an optimistic transaction model, which means the client will buffer all the writes within a transaction, and when the client calls commit function, the writes will be packed and sent to the servers. If there is no conflict, the writes which are the key-value pairs with specific version are written into the database, and can be read by other transactions.
+
+## Can I use TiKV as a key-value store?
+
+Yes. For now, we have a stable [TiKV driver for Go](https://github.com/pingcap/tidb/tree/master/store/tikv). There are also drivers for [other languages in development](../reference/clients/introduction.md).
+
+## How does TiKV compare to NoSQL databases like Cassandra, HBase, or MongoDB?
+
+TiKV is as scalable as NoSQL databases like to advertise, while including features like externally consistent distributed transactions and good support for stateless query layers such as TiSpark (Spark), Titan (Redis), or TiDB (MySQL)
+
+## What is the recommended number of replicas in the TiKV cluster? Is it better to keep the minimum number for high availability?
+
+Use 3 replicas for testing. If you increase the number of replicas, the performance declines but it is more secure. Whether to configure more replicas depends on the specific business needs.
+
+## If a node is down, will the service be affected? How long?
+
+TiKV uses Raft to synchronize data among multiple replicas (by default 3 replicas for each Region). If one replica goes wrong, the other replicas can guarantee data safety. Based on the Raft protocol, if a single leader fails as the node goes down, a follower in another node is soon elected as the Region leader after a maximum of 2 * lease time (lease time is 10 seconds).
+
+## Is the Range of the Key data table divided before data access?
+
+No. It differs from the table splitting rules of MySQL. In TiKV, the table Range is dynamically split based on the size of Region.
```

Comment:
I think current version does split by table by default.

Reply:
Can you suggest the text?

Reply:
@disksing Note these docs are TiKV specific, and don't concern ourselves with TiDB.